timepiece/templates/timepiece/contract/view.html

```diff
@@ -25,6 +25,11 @@
                 {% if perms.contracts.change_contract %}  {# FIXME: permission doesn't exist #}
                     <li><a href="{{ contract.get_admin_url }}">Edit</a></li>
                 {% endif %}
+                {% if perms.crm.view_project_time_sheet %}
+                    <li>
+                      <a href="{% url 'view_project_timesheet' object.id %}">View Time Sheet</a>
```

Comment:
Let's talk tomorrow, this implementation is not correct. The PMs want a link to the Hours Report directly accessible on a Project detail view.